package.json

```diff
@@ -105,6 +117,7 @@
     "ts-jest": "^29.1.0",
     "ts-loader": "^9.4.4",
     "typescript": "~5.5.3",
+    "vue": "^2.7.15",
```

Comment:
Should this be here? (It's also in peer dependencies)

Reply:
Yes. Its necessary for some dependencies during the build process. peer dependency only explains, that the leading app has to require it too,